emperor/util.py

```diff
@@ -532,3 +532,45 @@ def sanitize_mapping_file(data, headers):
 
     return out_lines[1::], out_lines[0]
 
+def guess_coordinates_files(dir_path):
+    """Given a directory return the file paths that can contain coordinates
+
+    Parameters
+    ----------
+    dir_path : str
+        path to the directory where coordinate files are contained
+
+    Returns
+    -------
+    list of str
+        list of filepaths pointing to the coordinates files
+
+    Notes
+    -----
+    If a path inside dir_path meets any of the following criteria, it will be
+    ignored:
+    - Is a hidden file
+    - Is named `Icon?`.
+    - Is folder
+    - Is part of the procrustes results from QIIME, see
+      transform_coordinate_matrices.py
+    """
+    coord_fps = []
+
+    for filepath in listdir(dir_path):
+        if filepath.startswith('.'):
+            continue
+        if filepath.startswith('Icon?') or filepath == 'Icon?':
```

Comment:
Yes you are correct, I've fixed this, thanks!

On (Feb-23-15| 0:09), Antonio Gonzalez wrote:
> > +    Notes
> > +    -----
> > +    If a path inside dir_path meets any of the following criteria, it will be
> > +    ignored:
> > +    - Is a hidden file
> > +    - Is named `Icon?`.
> > +    - Is folder
> > +    - Is part of the procrustes results from QIIME, see
> > +      transform_coordinate_matrices.py
> > +    """
> > +    coord_fps = []
> > +
> > +    for filepath in listdir(dir_path):
> > +        if filepath.startswith('.'):
> > +            continue
> > +        if filepath.startswith('Icon?') or filepath == 'Icon?':
> 
> Shouldn't startswith catch the second parameter too? I tested on ipython and it does ... 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/347/files#r25148291